docs/code-review.md

```diff
@@ -0,0 +1,330 @@
+# Okteto CLI code review comments
+
+- [Okteto CLI code review comments](#okteto-cli-code-review-comments)
+  - [What to look for in a code review?](#what-to-look-for-in-a-code-review)
+    - [Code review checklist](#code-review-checklist)
+    - [PR Description](#pr-description)
+    - [PR comments](#pr-comments)
+    - [Tests](#tests)
+      - [Unit tests](#unit-tests)
+      - [E2E tests](#e2e-tests)
+    - [Logs](#logs)
+    - [New dependencies](#new-dependencies)
+    - [Analytics](#analytics)
+  - [Common anti patterns](#common-anti-patterns)
+    - [Errors](#errors)
+      - [Don't Panic](#dont-panic)
+      - [Discarded errors](#discarded-errors)
+      - [Adding context](#adding-context)
+      - [Indent Error Flow](#indent-error-flow)
+    - [Naming convention](#naming-convention)
+      - [Variable Names](#variable-names)
+      - [Receiver's name](#receivers-name)
+      - [Be consistent](#be-consistent)
+      - [Search for an element](#search-for-an-element)
+
+## What to look for in a code review?
+
+### Code review checklist
+
+Here is a summary of the things every reviewer should be aware of while doing a CLI code review:
+
+- [ ] Does the PR have unit tests/e2e tests that covers all the scenarios?
+  *We should look forward to have all scenarios covered.*
+- [ ] Does the PR description explain what it does/solves?
+  *In the future we may need to know why the PR was created and what issues it solved, so this should be made clear in the description of the PR.*
+- [ ] Does it affect other services (actions/vscode plugin/pipeline/graphql/json logs)?
```

Comment:
Sometimes it is hard to know if some changes affect to other services or not. For example, how logs are going to be displayed in the UI (due to the stages, etc..) or the path stored within `filename` property of configmaps created by `okteto deploy` command. 

Maybe we can be more explicit on the things we should pay more attention because affect to other components